templates/dotnet-publish-image.yaml

```diff
@@ -0,0 +1,43 @@
+{{- $env_vars := list
+}}
+{{- $parameters := list
+    "PROJECT"
+    "IMAGE_NAME"
+    "USE_DOTNET_IMAGESTREAM_BASE_IMAGES"
+    "VERBOSITY"
+    "BUILD_PROPS"
+}}
+{{- $wrkspaces := list
+    "source"
+}}
+---
+apiVersion: tekton.dev/v1beta1
```

Comment:
Any reason to still use `v1beta1` ? `tekton.dev/v1` is out for a while now, and I think `v1beta1` is marked as "deprecated" for most objects (`Task`, …). So we should directly publish this in `v1` I think.

Reply:
This is from copy-paste. I'll update it to `tekton.dev/v1`.